kotlin/openID4VP/src/commonMain/kotlin/io/mosip/openID4VP/authorizationRequest/clientMetadata/Jwks.kt

```
@@ -29,15 +29,19 @@ object JwksSerializer : KSerializer<Jwks> {
-            key = "keys",
-            fieldType = "List<Jwk>",
-            deserializer = ListSerializer(Jwk.serializer()),
-            isMandatory = false
-        )!!
+
+        val keysElement = jsonObject["keys"]
+        val keys: List<Jwk> = if (keysElement is JsonArray) {
+            keysElement.mapNotNull { keyElement ->
+                try {
+                    jsonDecoder.json.decodeFromJsonElement(Jwk.serializer(), keyElement)
+                } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)